cv32/sim/uvmt_cv32/software.tcl

```diff
@@ -0,0 +1,27 @@
+# add fc execution trace
```

Comment:
Is this a file that Thales added?   I have removed a set of Tcl files and would prefer not to have them in here as they are tool-specific.